.circleci/config.yml

```diff
@@ -981,7 +981,9 @@ jobs:
       - slack-notify-failure-if-any
 
   build-docs:
-    executor: xcode-latest
+    executor:
```

Comment:
You don't need an executor root key. You can assign Xcode value to the `macos` key

Reply:
Do you mean like this?
``` 
build-docs: 
  macos:
    xcode: 13.4.1
```

Reply:
yes

Reply:
Great -- updated in latest